test/test_general.cpp

```diff
@@ -52,3 +53,65 @@ TEST(General, TestLineParsing) {
   EXPECT_EQ(data[1], "Entry2");
   EXPECT_EQ(data[2], "EscapedEntry1;EscapedEntry2");
 }
+
+TEST(General, Dijkstra) {
+  /*
+            (6)
+        .----------.
+       \/          \/
+  0 <-> 1 --> 2 <-> 3
+    (1)   (2)   (3)
+  */
+  CouplingMap cm = {{0, 1}, {1, 0}, {1, 2}, {2, 3}, {3, 2}, {1, 3}, {3, 1}};
+  Matrix      edgeWeights = {
```

Comment:
I think you can get around the clang-format error from the linter by inserting a newline in front of this line and reformatting. Always strange how two independent systems (clang-tidy locally, as well as pre-commit) say everything is alright, but a this system says no..

Reply:
That worked, thank you! I have been slowly running out of ideas how to appease the linter there

Reply:
Yeah. Sometimes it's a pain.. over time you develop a repertoire of tricks to fix these kind of issues 😌